Dockerfile

```diff
@@ -74,19 +74,23 @@ RUN apt-get -y install python-pip
 RUN pip install awscli
 RUN aws configure set default.s3.max_concurrent_requests 100
 COPY etc/generate_tiles.py /var/lib/postgresql/src/generate_tiles.py
-RUN chmod a+x /var/lib/postgresql/src/generate_tiles.py

```

Comment:
praise: appreciate this permissions cleanup!

Reply:
Thanks for actually commenting on it! Git's handling of permissions is weird and I always forget that it exists, so I wouldn't have thought to try optimizing that aspect of the Dockerfile it if you hadn't said something.